controls/update_def.cf.in

```diff
@@ -134,7 +134,7 @@ bundle common update_def
       # "masterfiles" shortcut.
 
       "mpf_update_policy_master_location" -> { "ENT-3692" }
-        string => ifelse( "am_policy_hub", "/var/cfengine/masterfiles",
+        string => ifelse( "am_policy_hub", "@prefix@/masterfiles",
```

Comment:
This is a weird one. A windows client for example would want to use /var/cfengine/ if it's bootstrapped to one of our packages, but during buildtime prefix would be C;\Program Files\Cfengine.

I think its OK in this context given the rest of how we build packages, just wanted to call it out.

Reply:
But windows is never a hub, right?

Reply:
*never* is such a strong word.
It's currently never an enterprise hub. Technically, there is nothing preventing it from being a policy hub. There can also be confusion if there is a mixed env where both our packages and FHS packages are used in the same envioronment. But I expect this to not be an issue for most cases, plus its a setting that can be adjusted via def.json.